s3/pkg/service/bucket.go

```diff
@@ -30,6 +30,17 @@ import (
 var ENC_KEY_LEN int = 32
 var ENC_IV_LEN int = 16
 
+var CRUDSupportedClouds = []string{"aws-s3", "azure-blob", "gcp-s3", "hw-obs"}
+
+func isCrudSupportedCloud(slice []string, item string) bool {
```

Comment:
just my opinion:
better implementation might be :
```
var CRUDSupportedClouds = map[string]struct{}{
                                   "aws-s3": struct{}{},
                                    "azure-blob": struct{}{},
                                    "gcp-s3": struct{}{}, 
                                    "hw-obs": struct{}{},
                               }
func isCrudSupportedCloud(cloud string) bool {
     _, ok := CRUDSupportedClouds[cloud]
     return ok
}

```

Reply:
+1